ansible/roles/docker-ce/tasks/main.yml

```diff
@@ -0,0 +1,49 @@
+- name: Install docker-ce
+  shell: |
+    sudo dnf config-manager --add-repo=https://download.docker.com/linux/centos/docker-ce.repo
+    sudo dnf install docker-ce --nobest -y
```

Comment:
I think we need to add install iptables as well but I'll have to try a clean run to verify. (When centos8 is the base OS)